notify/rocketchat/rocketchat.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2022 Prometheus Team
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rocketchat
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/go-kit/log"
+	"github.com/go-kit/log/level"
+	commoncfg "github.com/prometheus/common/config"
+
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/notify"
+	"github.com/prometheus/alertmanager/template"
+	"github.com/prometheus/alertmanager/types"
+)
+
+const maxTitleLenRunes = 1024
```

Comment:
Just out of interest where did you find this in the docs? I looked but couldn't see it.

Reply:
Oh, I just copied that from the slack notifier. As far as I can tell there is no explicit limit for the title-length in rocketchat. Only a limit for the complete message size set in the server configuration.
1k looks  kinda sane to me, but we could also drop it to the 250/256 set in pushover/discored notifiers.